### ISSUE TITLE
Version 2.2.0: Problem with the Chuffed solver

### ISSUE THREAD

**Issue by hakank** [MiniZinc/MiniZincIDE#55]
For all (or at least almost all) models that I've tried with the Chuffed solver (fzn-chuffed from the bin directory), it throws "=====ERROR=====". 

For example the very simple problem from minizinc-benchmarks/queens:
  $ minizinc queens.mzn 004.dzn --solver chuffed
"""
=====ERROR=====
"""

The MiniZinc IDE throws this message:
"""
Unknown error while executing the MiniZinc interpreter '/home/hakank/g12/MiniZincIDE/MiniZincIDE-2.2.0-bundle-linux/bin/fzn-chuffed': error code 1
"""

I'm running MiniZincIDE in Ubuntu 18.04LTS.

**Comment by soli** [comment]
Not really related (I think), but there's also a 'space' character missing in the mzn-chuffed script (last line) that leads to `Config exception: no solver with tag chuffed…` errors.

**Comment by Dekker1** [comment]
@hakank This seems to be a problem in with part of chuffed code in combination with the newer versions of GCC. A PR with a fix should be merged before the next MiniZinc patch.

@soli Thanks for the report. I've fixed the master version of the shell script.

**Comment by soli** [comment]
ty @Dekker1

**Comment by deluxerootit** [comment]
When on my Linux Ubuntu 20 I run: minizinc --solver Chuffed --solver-statistics -O1 --time-limit 2000 model.mzn 

I get the following error: Confan you help me?

**Comment by deluxerootit** [comment]
Also when I import Instance from z3 I get the following error :
ImportError: cannot import name 'Instance' from 'minizinc' (/home/maryam/.local/lib/python3.8/site-packages/minizinc/__init__.py)

Any idea?

**Comment by Dekker1** [comment]
@deluxerootit You are commenting on an old issue that is likely unrelated `=====ERROR=====` is the standard output by MiniZinc when the solver does not exits correctly or has unexpected output. Please open a separate issue with more details. The issue would probably be most appropriate for the MiniZinc/libminizinc repository.

The second issue seems different again. This might just be that you have a script yourself that is also named `minizinc`, but could be another issue. Again more details would be required. This issue would then be most appropriate for the MiniZinc/minizinc-python repository
